.snyk

```diff
@@ -7,4 +7,9 @@ ignore:
         reason: No upgrade available
         expires: 2024-07-18T00:00:00.000Z
         created: 2023-07-18T12:52:43.840Z
+  SNYK-JS-INFLIGHT-6095116:
+    - '*':
+        reason: No upgrade available
+        expires: 2024-12-01T00:00:00.000Z
```

Comment:
suggestion: only wait for a quarter to see if a fix is provided

Reply:
revised to a quarter